docs/unison-sample.prf

```diff
@@ -0,0 +1,39 @@
+# Unison preferences file
+
+# The local path to jetpack
+# This can be adjusted to a specific sub-path of the Jetpack repo (like jetpack-mu-wpcom)
+root = /local/path/to/jetpack
+force = /local/path/to/jetpack
+
+# This is the remote host you will be syncing changes to.
+root = ssh://username@remote-host.com//path/to/jetpack/on/remote
+
+batch = true
+
+# Ignore common IDE files
```

Comment:
We should probably prevent mid-build/working/temp files from being synced - this is mostly true for JS(X) and TS(X) files that  get compiled and consolidated during the build process, as that likely occurs in some sort of temporary directory that we wouldn't want to sync to avoid lots of unnecessary changes.